src/context/themeContext.js

```diff
@@ -1,17 +1,16 @@
 import React, { Component } from 'react';
+import { saveState, loadState } from '../DataService/localDataService';
 const ThemeContext = React.createContext();
 
 class ThemeContextProvider extends Component {
   state = {
-    theme: 'dark'
+    theme: 'light'
   };
 
   toggleTheme = () => {
-    const darkOS = window.matchMedia('(prefers-color-scheme: dark)').matches;
-
     this.setState((prevState) => {
       return {
-        theme: prevState.theme === 'light' ? 'dark' : 'light'
+        theme: prevState.theme === 'dark' ? 'light' : 'dark'
```

Comment:
do you still need theme provider after you created theme reducer?

Reply:
fixed